tests/jobs/scala/src/main/scala/DatasetGenerator.scala

```diff
@@ -0,0 +1,57 @@
+import org.apache.spark.sql.SparkSession
```

Comment:
let's create a separate `sorting` package for both generator and sorter with a README so they're not mixed up with the rest of the testing jobs